src/components/menu-list/menu-list-renderer.tsx

```
@@ -124,160 +89,57 @@ export class MenuListRenderer {
         if ('separator' in item) {
             return (
                 <li class="mdc-deprecated-list-divider" role="separator">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<hr>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)